Online/Entity/OnlinePhysicalObject.cs

```diff
@@ -343,5 +343,14 @@ public static void HitByExplosion(OnlinePhysicalObject objectHit, float hitfac)
         {
             objectHit?.apo.realizedObject?.HitByExplosion(hitfac, null, 0);
         }
+
+
+        [RPCMethod]
+        public static void ScavengerBombExplode(OnlinePhysicalObject scavBomb)
+        {
+
+            (scavBomb.apo.realizedObject as ScavengerBomb)?.Explode(null);
```

Comment:
You might want to pass in the exact pos of the bomb when it explodes so that the location of where it explodes is EXACTLY the same. Otherwise lag might cause a bomb to explode earlier for receiving clients